MN/Algoritmi/Algoritmi-ad-hoc/IsDiagonallyDominant.m

```diff
@@ -0,0 +1,28 @@
+% checks if a matrix A is diagonally dominant 
+
+function is_diag_dom = IsDiagonallyDominant(A)
+
+	[n n] = size(A);
+	is_diag_dom = 1; % I assume it is diagonally dominant
+
+	for i = 1 : n
+		if A(i, i) < 0
```

Comment:
if-ul asta e gresit. scoate-l.
A diag dominanta daca:
|aii| > sum |aij| cu j != i

Reply:
Ah, da, e gresit pentru diagonal dominanta. Dar am notat in caiet ca pentru Cholesky, elementele de pe diagonala principala trebuie sa fie pozitive, deci o sa adaug verificarea in Cholesky.m.

E ok asa?